commands/util.go

```diff
@@ -65,11 +65,17 @@ func driversForNodeGroup(ctx context.Context, dockerCli command.Cli, ng *store.N
 			return nil, errors.Errorf("failed to find driver %q", f)
 		}
 	} else {
+		// empty driver name is explicit to allow fallbacking to a
+		// docker-container driver with older daemon that don't support
+		// buildkit (< 18.06) for backward compatibility.
 		ep := ng.Nodes[0].Endpoint
 		dockerapi, err := clientForEndpoint(dockerCli, ep)
 		if err != nil {
 			return nil, err
 		}
+		if _, err = dockerapi.Ping(ctx); err != nil {
```

Comment:
Comment: checking that connection is available is needed to determine the driver type. If this fails then can't continue with driver selection.